### PR TITLE
Drive tuning

### PR DESCRIPTION
**Pull request by KSatsujin**

### CODE REVIEW

File: src/main/java/frc/robot/Dashboard.java [hunk 1]
```diff
@@ -28,52 +30,82 @@ public class Dashboard extends SubsystemBase {
...
+                                .add("Rotation Speed Limit", Constants.DrivebaseConstants.DEFAULT_ROTATION_SPEED_LIMIT)
+                                .withWidget(BuiltInWidgets.kNumberSlider)
+                                .withProperties(Map.of("min", 0, "max", 1))
+                                .getEntry();
+
+                telemetryLayout = Shuffleboard.getTab(Constants.ShuffleboardConstants.GAME_TAB_NAME)
+                                .getLayout("Telemetry", BuiltInLayouts.kGrid)
+                                .withProperties(Map.of("Number of columns", 2, "Number of rows", 1))
+                                .withSize(9, 5)
+                                .withPosition(6, 0);
```
Comment (shanealv) [inline src/main/java/frc/robot/Dashboard.java:88]: Most of these positions assume a default size of 64.  This should be communicated in the README or something

File: src/main/java/frc/robot/commands/Drive.java [hunk 2]
```diff
@@ -40,15 +40,15 @@ public Drive(DrivebaseSubsystem drivebase, DoubleSupplier translateX, DoubleSupp
   @Override
   public void execute() {
     // Reversed strafe and rotation axis
-    double x = getDeadzone(-translateX.getAsDouble()) * getSpeedMultiplier();
-    double y = getDeadzone(translateY.getAsDouble()) * getSpeedMultiplier();
-    double z = getDeadzone(-rotateZ.getAsDouble());
+    double x = getDeadzone(-translateX.getAsDouble()) * getSpeedMultiplier() * Dashboard.maxSpeedEntry.get().getDouble();
+    double y = getDeadzone(translateY.getAsDouble()) * getSpeedMultiplier() * Dashboard.maxSpeedEntry.get().getDouble();
```
Comment (shanealv) [inline src/main/java/frc/robot/commands/Drive.java]: You already use `Dashboard.maxSpeedEntry.get().getDouble()` in `getSpeedMultiplier()`, so you are applying the speed multiplier twice.

**Review by shanealv — CHANGES_REQUESTED** [review]
Two bits of feedback for you.  The UI looks good though!

File: src/main/java/frc/robot/Dashboard.java (hunk 1 above)
Comment (KSatsujin) [reply src/main/java/frc/robot/Dashboard.java:88]: Will do!

File: src/main/java/frc/robot/commands/Drive.java (hunk 2 above)
Comment (KSatsujin) [reply src/main/java/frc/robot/commands/Drive.java]: I just realized that.

**Comment by KSatsujin** [comment]
I fixed all of the requested changes.

**Review by shanealv — APPROVED** [review]

**Pull request merged**